ArticleTemplates/liveblogTemplate.html

```diff
@@ -149,6 +149,8 @@ <h1 class="the-minute__headline selectable">__TITLE__</h1>
             <img src="__TEMPLATES_DIRECTORY__assets/img/loading.gif" alt="Loading…"/>
         </div>
 
+        __CONTRIBUTIONS_EPIC__
```

Comment:
On further thinking I see why we would want this to be between the blocks instead of at the end so I have moved it. I certainly always thought it was a the end though.